tensorflow_app/README.md

```diff
@@ -1,6 +1,6 @@
 # Cloud Native Video Sreaming and Analytics
 
-To conplete video streaming and analytics three containers are used. 
+To complete video streaming and analytics three containers are used. 
```

Comment:
Same as above: This is fine for this commit, but in general you want all of the changes in a commit to be related. And if there are typos in documentation or comments like this, it's better to have them as a separate commit so that they are super easy to review, approve and merge.